modules/client-configuration/pages/cobbler.adoc

```diff
@@ -206,6 +207,27 @@ Doing so can cause errors.
 Always use [command]``/usr/sbin/spacewalk-service`` to start or stop {productname}.
 ====
 
+////
+=== Background Information about the Synchronization Process
+
+// Taken from https://bugzilla.suse.com/show_bug.cgi?id=1141603
+// To be edited!!!
+
+A sync is basically a rebuild of every file cobbler touched. I will try give you an impression with an numbered list.
+
+1. Run Pre-Sync-Triggers (this can be any number of shell scripts to my knowledge)
+2. Delete all files and directories which are not allowed in /var/www/cobbler
+3. Create all needed folders
+4. Delete all elements inside the folders.
+5. Generate the tftpd directory
+6. Write the DHCP files (if management is enabled)
+7. Do the same with DNS
+8. Clean the Cache
+9. If rsync management is enabled do that
+10. Run Post-Sync-Triggers (this can also be any number of shell scripts to my knowledge)
+
+////
+
```

Comment:
I would classify this as a concept. For that reason alone, I would recommend rewording this into prose, and put it at the top of the file.

Reply:
ok, done.